backend/src/bundles/subscription/subscription.model.ts

```diff
@@ -0,0 +1,46 @@
+import { type RelationMappings, Model } from 'objection';
+
+import {
+    AbstractModel,
+    DatabaseTableName,
+} from '~/common/database/database.js';
+
+import { SubscriptionPlanModel } from '../stripe-events/models/subscription-plan.model.js';
+import { UserModel } from '../users/user.model.js';
+
+class SubscriptionModel extends AbstractModel {
+    public 'status': string;
```

Comment:
`enum SubscriptionStatus {
    ACTIVE = 'active',
    CANCELED = 'canceled',
    TRIALING = 'trialing',
    PAST_DUE = 'past_due',
    INCOMPLETE = 'incomplete', 
    INCOMPLETE_EXPIRED = 'incomplete_expired',
    UNPAID = 'unpaid',
}`

perhaps enum could be better

Reply:
I can't use enum, because status'll be changed dynamically by stripe events.

Reply:
Anyway, status has discrete values.